README.md

```diff
@@ -63,7 +63,7 @@ $ cat /usr/share/ramalama/shortnames.conf
 Ramalama is available via PyPi [https://pypi.org/project/ramalama](https://pypi.org/project/ramalama)
 
 ```
-pip install ramalama
+pipx install ramalama
```

Comment:
I am fine with this, but I have not found pipx working well on my Fedora system.

Reply:
😢